### PR TITLE
v21.1.7 release notes

### PR DESCRIPTION
**Pull request by ericharmeling**
Fixes https://github.com/cockroachdb/docs/issues/10714.

Related to https://github.com/cockroachdb/cockroach/issues/68428.

### CODE REVIEW

File: releases/v21.1.7.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    <a href="https://binaries.cockroachdb.com/cockroach-v21.1.7.src.tgz"><button id="source" class="filter-button" data-scope="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+<section class="filter-content" data-scope="windows">
+{% include windows_warning.md %}
+</section>
+
+### Docker image
+
+{% include copy-clipboard.html %}
```
Comment (ianjevans) [inline releases/v21.1.7.md:34]:
```suggestion
{% include_cached copy-clipboard.html %}
```

File: releases/v21.1.7.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+- Previously, CockroachDB could encounter an internal error or crash when performing a cast of [`NULL`](../v21.1/null-handling.html) [`JSON`](../v21.1/jsonb.html) value to [Geography or Geometry types](../v21.1/spatial-data.html). Now this is fixed. [#67902][#67902]
+- [`INSERT`](../v21.1/insert.html) and [`UPDATE`](../v21.1/update.html) statements which operate on larger rows can now be split into batches using the `sql.mutations.mutation_batch_byte_size` setting. [#67958][#67958]
+- A rare bug that could result in a crash while creating a [debug.zip file](../v21.1/cockroach-debug-zip.html) has been fixed. The bug was only possible to hit if a debug.zip file was captured during a period of rapid lease movement. [#67728][#67728]
+- Previously the [`GRANT`](../v21.1/grant.html) and [`REVOKE`](../v21.1/revoke.html) commands would incorrectly handle role names. CockroachDB treats role names as case insensitive, but these commands were incorrectly handling the names. Now, `GRANT` and `REVOKE` normalize the names and are case-insensitive. [#67901][#67901]
```
Comment (ianjevans) [inline releases/v21.1.7.md:84]:
```suggestion
- Previously the [`GRANT`](../v21.1/grant.html) and [`REVOKE`](../v21.1/revoke.html) commands would incorrectly handle role names. CockroachDB treats role names as case-insensitive, but these commands were incorrectly handling the names. Now, `GRANT` and `REVOKE` normalize the names and are case-insensitive. [#67901][#67901]
```

**Review by ianjevans — APPROVED** [review]
Just two small nits. Looks good.

**Review by ericharmeling — COMMENTED** [review]
TFTR @ianjevans !

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/11021#-:-MggNGjz7hTV0NWEHb01:b-vo6kmr)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @ianjevans and @mgartner)

___
*[releases/v21.1.7.md, line 34 at r1](https://reviewable.io/reviews/cockroachdb/docs/11021#-MgfYtApzAzyXS1q2jcY-r1-34:-MggN9YNA6tB1CsPEUke:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/660d5e30b96cb43f3d9d10de5d7d6d1300fab110/releases/v21.1.7.md#L34)):*
<details><summary><i>Previously, ianjevans (Ian Evans) wrote…</i></summary><blockquote>

```suggestion
{% include_cached copy-clipboard.html %}
```
</blockquote></details>

Done.
___
*[releases/v21.1.7.md, line 84 at r1](https://reviewable.io/reviews/cockroachdb/docs/11021#-MgfYtApzAzyXS1q2jcY-r1-84:-MggNG1p8X20uSXuKQ3_:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/660d5e30b96cb43f3d9d10de5d7d6d1300fab110/releases/v21.1.7.md#L84)):*
<details><summary><i>Previously, ianjevans (Ian Evans) wrote…</i></summary><blockquote>

```suggestion
- Previously the [`GRANT`](../v21.1/grant.html) and [`REVOKE`](../v21.1/revoke.html) commands would incorrectly handle role names. CockroachDB treats role names as case-insensitive, but these commands were incorrectly handling the names. Now, `GRANT` and `REVOKE` normalize the names and are case-insensitive. [#67901][#67901]
```
</blockquote></details>

Done.

**Pull request merged**